transiXNOR/plotting.py

```diff
@@ -183,26 +221,23 @@
 	ax.xaxis.set_minor_locator(AutoMinorLocator())
 	# plt.xlim([-0.1, 0.3])
 	plt.ylim([1e-3, 1e3])
-	plt.savefig(model_path + '/plots/current_vds_' + Vds + '_vbg_0.3.pdf',
+	plt.savefig(model_path + '/plots/current_vds_' + '%.2f' % (Vds) + '_vbg_0.3.pdf',
 				bbox_inches='tight', transparent=True)
 	plt.clf()
 
 if (COMBINE_CURRENT_VIA_SYMMETRY):
-	vdsmin=-0.10; vdsmax=0.3; vdsN=41;
-	vbgmin=-0.10; vbgmax=0.3; vbgN=41;
-	vtgmin=-0.10; vtgmax=0.3; vtgN=41;
+	vdsmin=-10; vdsmax=30; vdsN=41;
```

Comment:
Add necessary comments here to explain vds, vtg, vbg are scaled up by 100 times.